link_shortener/templates.py

```diff
@@ -12,5 +14,6 @@ def template_loader(template_file, *args, **kwargs):
     file_loader = PackageLoader(__name__, 'templates')
     env = Environment(loader=file_loader)
     template = env.get_template(template_file)
-    output = template.render(*args, **kwargs)
+    ga_id = config('GOOGLE_ANALYTICS_ID')
```

Comment:
Není to spíš DSN než ID?

Reply:
A ga_id prosím přejmenuj na analytics_dsn, u ga mi nikdy nedojde co to je 😄

Reply:
Google to vola "Tracking ID" :)
https://support.google.com/analytics/answer/7372977?hl=en
inak nezalezi mi na nazve , takze pokial chces tak kludne dam vsade `analytics_dsn`. Chces to aj v kode aj v env, jo?

Reply:
Tak to nazvi podle nich, v kódu lower case, v configu upper

Reply:
9ec275cf79802b1c4da9f79082e9df7f31cf8484